### ISSUE TITLE
[Bug]: Double reserve a table crashes the reservation-listing

### ISSUE THREAD

**Issue by git-gnulf** [tastyigniter/ti-ext-reservation#27]
### What happened?

I tried what happens if I reserve a specific table via admin-interface at a date and time it is already reserved. I forced a conflict and want to see what TI says. The reservation was saved without any error. 
But after saving the page reload and jumped to start-page (my-restaurant.com/) - left admin-interface!
I can go back to admin-interface. But if I click at Sales->Reservations I was kicked out from admin and jumped to start-page.

### What did you expect to happen?

I expect, that TI checked if a chosen table could be reserve or no. If not, TI should show a warning/error and should not save. ... and should not kick the user out ;)

Steps to reproduce:
- condition: Automatically Allocate Tables must be ON
- let a customer create a reservation.
- remember the date and time
- go to Sales -> Reservation -> Listing View
- have a look at the entry. Check which table is allocated.
- In Admin-Interface click at the New-Button at top-left and create manually a table-reservation
- fill out the date and time you remember from the first reservation
- chose the same table -> and force a conflict
- click the save-button

You should now see for about 1/2 sec a redirect and be at your start-page.

My TI-Version: 3.1.2

### Version

Other (please detail)

### Relevant log output

```shell
Error: Class "Symfony\Component\Debug\Exception\FatalThrowableError" not found
in /doc-root/app/system/traits/ViewMaker.php:233
```

**Comment by git-gnulf** [comment]
... oh, is it possible to simple delete the last reservation in database via mysql? Is the reservation stored only in one table or is it a complex structure? ... my TI is blocked now and I have to unblock it ...

**Comment by git-gnulf** [comment]
@sampoyigi, @ryanmitchell have you seen this issue? Have you an idea / hint for me, what I can do to fix my broken view of the reservations?

**Comment by git-gnulf** [comment]
@sampoyigi, @ryanmitchell:  Good news! I assume, I have a part of a solution. I dived into the database and looked at the table "ti_reservations". In column "duration" I found no value but NULL. If I changed "duration" to a regular value (for example 120) the reservation-listing in admin-backend works again. Hooray! :smile: 

When I create the manual reservation - in admin-backend - I leave the field "reservation duration" blank cause the description-comment says: "Leave blank to use the default stay time."

But It seems the default value was not caught and stored. 
At this point I m out. I dont know where I can find the corresponding code. Now it's your part :grin:
